src/containers/Explorer/FilterNav.js

```diff
@@ -107,8 +168,12 @@ const FilterNavComponent = props => {
           value={categoryFilters}
           labelKey="name"
           valueKey="normalized_name"
-          onChange={values => addCategoryFilter(values[values.length - 1])}
-          onClose={removeCategoryFilter}
+          onChange={values => {
+            if (values.length > categoryFilters.length) {
+              addCategoryFilterAction(values[values.length - 1]);
```

Comment:
I've been wondering about the best way to do this, and if there were any changes since ES6. Ran some tests on this site and it came back with slicing a new array being significantly faster. Might be worth adopting as a general guideline in our repo? Typically it'd happen in 1 step like so: 
```
const [last] = [1, 3, 4, 5].slice(-1)
```

![screen shot 2018-08-27 at 23 08 17](https://user-images.githubusercontent.com/1154150/44698660-e327af80-aa4e-11e8-9dd5-a3aff3ed73d4.png)

Reply:
I actually think .slice(-1) also reads cleaner... All for it.